deploy/ingress-nginx/kustomization.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ingress-nginx-helmrepo.yaml
+- ingress-nginx.yaml
+- ingress-nginx-namespace.yaml
+- ingress-nginx_vpa.yaml
```

Should we name the files consistently?
```suggestion
- ingress-nginx-vpa.yaml
```